cli/tests/activate.bats

```diff
@@ -2487,6 +2487,58 @@ EOF
   assert_success
 }
 
+@test "profile: JUPYTER_PATH not modified when Jupyter is not installed" {
+  project_setup
+
+  # Shouldn't be set by default.
+  run "$FLOX_BIN" activate -- \
+    echo JUPYTER_PATH is '$JUPYTER_PATH'
+  assert_success
+  assert_output "JUPYTER_PATH is"
```

Comment:
Shouldn't there be an extra space after `is` in this assertion since it's matching the whole output?

Reply:
And shouldn't it fail if it doesn't match that exactly e.g. shouldn't this test be failing?

Reply:
The unset variable gets evaluated to an empty string and gets dropped as an argument somewhere in parsing:
```
% ft activate -- echo foo '$NULL' | cat -e
foo$
% bash -c 'echo foo $NULL' | cat -e 
foo$
```

I've pushed an additional commit which wraps them in `bash -c` and double quotes to make it clearer.